ValueError: zero length field name in format

```
$ koji list-tag-extras product-redacted-build
Traceback (most recent call last):
  File "/usr/lib/python2.6/site-packages/kojismokydingo/cli/__init__.py", line 349, in __call__
    return self.handle(options) or 0
  File "/usr/lib/python2.6/site-packages/kojismokydingo/cli/tags.py", line 711, in handle
    json=options.json)
  File "/usr/lib/python2.6/site-packages/kojismokydingo/cli/tags.py", line 677, in cli_list_tag_extras
    quiet=quiet)
  File "/usr/lib/python2.6/site-packages/kojismokydingo/cli/__init__.py", line 218, in tabulate
    print(fmt.format(*headings), file=out)
ValueError: zero length field name in format
2020-08-25 08:13:01,117 [ERROR] koji: ValueError: zero length field name in format
```

--- a/kojismokydingo/cli/__init__.py
+++ b/kojismokydingo/cli/__init__.py
@@ -211,8 +211,9 @@ def tabulate(headings, data, key=None, sorting=0,
                   zip_longest(widths, headings)]
 
     # now we create the format string based on the max width of each
-    # column plus some spacing
-    fmt = "  ".join("{:<%i}" % w for w in widths)
+    # column plus some spacing. Note that python 2.6 mandates the
+    # field index be specified, so we MUST use enumerate here.
+    fmt = "  ".join("{%i:<%i}" % iw for iw in enumerate(widths))
 
     if headings and not quiet:
         print(fmt.format(*headings), file=out)

